low-level/files/opt/seagate/sspl/setup/sspl_reset.py

```diff
@@ -42,15 +42,19 @@ def process(self):
         CMD = "id -u sspl-ll"
         output, error, returncode = SimpleProcess(CMD).run()
         if returncode != 0:
-            raise SetupError(returncode, "ERROR: %s - CMD %s", error, CMD)
+            msg = "ERROR: %s - CMD %s", error, CMD
+            logger.error(msg)
+            raise SetupError(returncode, msg)
         else:
             self.user_present=True
 
         if self.user_present:
             CMD="/usr/sbin/userdel sspl-ll"
             output, error, returncode = SimpleProcess(CMD).run()
             if returncode != 0:
-                raise SetupError(returncode, "ERROR: %s - CMD %s", error, CMD)
+                msg = "ERROR: %s - CMD %s", error, CMD
```

Comment:
This should be `msg = "ERROR: %s - CMD %s" %(error, CMD)`

Reply:
done.